File: problems/prime-numbers/boilerplate-full/function.js

  ##USER_CODE_HERE##
  
  const input = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n').join(' ').split(' ');
  const start = parseInt(input.shift());
  const end = parseInt(input.shift());
  const result = findPrimes(start, end);
  console.log(result);
      

File: problems/prime-numbers/boilerplate-full/function.rs

  use std::io::{self, BufRead};
  
  ##USER_CODE_HERE##
  
  fn main() {
    let stdin = io::stdin();
    let mut input = stdin.lock().lines().map(|l| l.unwrap());
    let start: i32 = input.next().unwrap().parse().unwrap();
  let end: i32 = input.next().unwrap().parse().unwrap();
    let result = findPrimes(start, end);
    println!("{}", result);
  }
      